### DISCUSSION TITLE
Display special characters on 5250 Terminals

### DISCUSSION THREAD

**Discussion by AMDOCA** [codefori/vscode-ibmi#1966] [IBM i] [answered]
Hi fellows,
I would like to correctly display special characters like "ñ", "á", etc. on 5250 terminals. I've tried Terminal Encoding Parameter with no success, Vs Code keeps up replacing these characters by "?", maybe someone has a better idea.

Please see an example:
 
![5250 Sample Screen](https://github.com/codefori/vscode-ibmi/assets/79931072/122f502b-fa32-4476-b4b2-fbd85756fa6c)

**Comment by worksofliam** [comment]
Did you set the terminal CCSID inside the Connection Settings?

**Reply by AMDOCA** [reply]
Yes, I did that.
![image](https://github.com/codefori/vscode-ibmi/assets/79931072/e34f244c-f99a-460d-b3b8-c7d64e62cae3)

**Reply by worksofliam** [reply]
What did you set the new value to?

**Reply by AMDOCA** [reply]
Actually, I've tried almost all of them.
![image](https://github.com/codefori/vscode-ibmi/assets/79931072/71c5111d-f640-4f3d-ab7d-52681c14472b)

**Reply by worksofliam** [reply]
If you're having no luck there, I'd see this page: https://codefori.github.io/docs/tips/ccsid/

**Reply by AMDOCA** [reply, marked as answer]
After reading the page provided by you, I found a possible solution to resolve this issue. It takes to change an AS/400's environment variable and restart the ssh daemon. I should find someone who help me with it. Thank you for your attention to this matter.
